EventFilter/CSCRawToDigi/python/cscUnpacker_cfi.py

```diff
@@ -1,9 +1,9 @@
 import FWCore.ParameterSet.Config as cms
 
 # Import from the generic cfi file for CSC unpacking
-import EventFilter.CSCRawToDigi.muonCSCDCCUnpacker_cfi
+from EventFilter.CSCRawToDigi.muonCSCDCCUnpacker_cfi import muonCSCDCCUnpacker
+muonCSCDigis = muonCSCDCCUnpacker.clone()
 
-muonCSCDigis = EventFilter.CSCRawToDigi.muonCSCDCCUnpacker_cfi.muonCSCDCCUnpacker.clone()
 # Define input to the unpacker
 muonCSCDigis.InputObjects = cms.InputTag("rawDataCollector")
```

Comment:
since this file is modified, please move the parameters inside `.clone` and  drop the type specifications for all existing parameters. The reasons are
- less verbose implementation
- dropped type specifications improve syntax checks to avoid accidentally inserting a parameter where a change is needed
```suggestion
muonCSCDigis = muonCSCDCCUnpacker.clone(
  # Define input to the unpacker
  InputObjects = "rawDataCollector",
```

Reply:
Sure. That would shorten it to 
```python
# Import from the generic cfi file for CSC unpacking
from EventFilter.CSCRawToDigi.muonCSCDCCUnpacker_cfi import muonCSCDCCUnpacker

muonCSCDigis = muonCSCDCCUnpacker.clone(
    # This mask is needed by the examiner
    ExaminerMask = 0x1FEBF7F6
)

```